ubuntudesign/snapcraft.io
# Restructure includes into shared layout

This is partly to solve an issue with how to split the two parts of Google Tag Manager markup into includes.

But more generally, I believe consolidated wrapping templates are much simpler to read than many includes.

QA
---

- Check site still builds and looks fine in both docs and main site pages
- Check Google Tag Manager code is included as expected in both docs and main site pages
